src/js/components/DateTimeDrop.js

```diff
@@ -1,42 +1,52 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import classnames from 'classnames';
-import moment from 'moment';
-import Box from './Box';
-import Header from './Header';
-import Title from './Title';
-import Button from './Button';
-import LinkPreviousIcon from './icons/base/LinkPrevious';
-import LinkNextIcon from './icons/base/LinkNext';
-import AddIcon from './icons/base/Add';
-import SubtractIcon from './icons/base/Subtract';
-import CSSClassnames from '../utils/CSSClassnames';
-import { announce } from '../utils/Announcer';
-import Intl from '../utils/Intl';
-import KeyboardAccelerators from '../utils/KeyboardAccelerators';
+import React, { Component } from "react";
```

Comment:
any reason why we are switching everything to double quotes? Grommet pattern is to use single quote.